glossary.html

```diff
@@ -5,8 +5,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="utf-8">
     <link rel="stylesheet" href="styles/normalize.css">
-    <link rel="stylesheet" href="styles/mobile.css"   media="screen and (max-width: 768px)">
     <link rel="stylesheet" href="styles/main.css">
+    <link rel="stylesheet" href="styles/mobile.css"   media="screen and (max-width: 768px)">
```

Comment:
Your responsive styles in media queries should come AFTER the default styles, otherwise the default styles will override your responsive styles